src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -973,10 +1008,10 @@ static void Main()
             comp.VerifyDiagnostics(
                 // (6,13): error CS0023: Operator '!' cannot be applied to operand of type 'method group'
```

Comment:
>// (6,13): error CS0023: Operator '!' cannot be applied to operand of type 'method group' [](start = 16, length = 89)

Here is another candidate for using new error message, consider opening an issue.